main.go

```diff
@@ -28,7 +35,38 @@ func main() {
 		cancel()
 	}()
 
-	go http.ListenAndServe(":80", api.New())
+	// Load Environment Variables
+	envConf, err := FromEnv()
+	if err != nil {
+		log.Fatalf("Error getting environment variables %s", err)
+	}
+
+	dd := dogstatsd.New("wattpad.ship-it.", logger)
+	go dd.SendLoop(time.NewTicker(time.Second).C, "udp", envConf.DataDogAddress())
+	hist := dd.NewTiming("consumer.time", 1.0).With("consumer", "ship-it-consumer", "queue", envConf.QueueName)
```

Comment:
For consistency with other service's queue consumer metrics, this should be `worker.time`. Since this service will have multiple workers, we should use `worker:ecrconsumer` instead of `ship-it-consumer`.